netty-server/docker-compose.yml

```diff
@@ -62,8 +68,23 @@ services:
 
   dynamodb:
     image: amazon/dynamodb-local:1.24.0
-    command: "-jar DynamoDBLocal.jar -sharedDb -dbPath /data"
+    command: "-jar DynamoDBLocal.jar -sharedDb -dbPath ./data"
     ports:
       - "8000:8000"
+    user: root
+    volumes:
+      - ./formkiq/dynamodb:/home/dynamodblocal/data
+    working_dir: /home/dynamodblocal
+
+  keycloak:
+    image: quay.io/keycloak/keycloak:26.0.6
```

Comment:
Image version is fixed, since file format with configuration can change in the future, or there might be other breaking changes introduced.